flybywiresim/aircraft
# feat: suppress cabin announcements when no pax

Fixes #8298

## Summary of Changes

- Added logic which disables all cabin sounds when the current flieght hasn´t any passangers
- Added "globalSettings" redux for storing data global in the "EFB" project without using the simvars. I need this for storing the sound states the player defines on the settings page
    

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): louisdev08

## Testing instructions

1. Start a flight
2. Make sure there are passengers on Board and the cabin crew sounds (announcements, passenger ambience and boarding music) is enabled in the settings page of the EFD.
3. Switch on beacon lights or do another event which triggers a cabin crew announcement
4. While the flight attendant is doing the announcement, switch the pax to 0 in the EFB
5. When pax gets 0 the attendant immediately stops doing the announcement.



## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
1. Build the **A32NX**
1. Install the **A32NX**
